pkg/api/converterfromupgradeapi.go

```diff
@@ -10,18 +13,38 @@ import "github.com/Azure/acs-engine/pkg/api/vlabs"
 // for converting.
 ///////////////////////////////////////////////////////////
 
-// ConvertUpgradeContainerServiceToVLabs converts an unversioned ContainerService to a vlabs ContainerService
-func ConvertUpgradeContainerServiceToVLabs(api *UpgradeContainerService) *vlabs.UpgradeContainerService {
-	vlabsUCS := &vlabs.UpgradeContainerService{}
-	vlabsUCS.OrchestratorProfile = &vlabs.OrchestratorProfile{}
-	convertUpgradeOrchestratorProfileToVLabs(api.OrchestratorProfile, vlabsUCS.OrchestratorProfile)
-	return vlabsUCS
+// ConvertUpgradeContainerServiceToV20170930 converts an unversioned UpgradeContainerService to a v20170930 UpgradeContainerService
+func ConvertUpgradeContainerServiceToV20170930(api *UpgradeContainerService) *v20170930.UpgradeContainerService {
```

Comment:
Based on previous discussion wasn't this going to be removed?

Reply:
addressed